** (emacs-version) has been modified to indicate Debian modifications.
   Provided by: Rob Browning <rlb@defaultvalue.org>

---
 lisp/version.el |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/lisp/version.el
+++ b/lisp/version.el
@@ -56,8 +56,8 @@
   (interactive "P")
   (let ((version-string
          (format (if (not (interactive-p))
-		     "GNU Emacs %s (%s%s%s)\n of %s on %s"
-		   "GNU Emacs %s (%s%s%s) of %s on %s")
+		     "GNU Emacs %s (%s%s%s)\n of %s on %s, modified by Debian"
+		   "GNU Emacs %s (%s%s%s) of %s on %s, modified by Debian")
                  emacs-version
 		 system-configuration
 		 (cond ((featurep 'motif)
